pkg/apis/kops/cluster.go

```diff
@@ -40,67 +40,72 @@ type ClusterList struct {
 
 type ClusterSpec struct {
 	// The Channel we are following
-	Channel string `json:"channel,omitempty"`
+	Channel               string `json:"channel,omitempty"`
```

Comment:
Yep! Fixed in https://github.com/kubernetes/kops/pull/694/commits/07eb92fd436b445cc37d02aa26030676bc1b251a